.gitattributes

```diff
@@ -1,6 +1,5 @@
 # `git apply` and friends don't understand CRLF, even on windows. Force those
 # files to be checked out with LF endings even if core.autocrlf is true.
-*.patch text eol=lf
```

Comment:
We shouldn't remove this imo - it is more likely to cause further issues in future patch changes.

Reply:
Unfortunately in order to apply the reland_lzma_sdk_update_to_24_09.patch this change is needed.  Given that 32 is soon entering end of life, it is better to land this patch.